assets/js/modules/analytics/components/common/AdsConversionIDTextField.js

```diff
@@ -78,7 +85,7 @@ export default function AdsConversionIDTextField() {
 	// but only hide it if the value is valid otherwise the user will be blocked.
 	if (
 		isValidValue &&
-		! snippetEnabled &&
+		! ( hasUAProperties && snippetEnabled ) &&
```

Comment:
Thanks @aaemnnosttv, great spot. Taking a second look it appears that the `enableUA` flag is sufficient to use instead of the check for UA properties and will cover both cases. I've pushed this change.